modules/postfix/manifests/postmapfile.pp

```diff
@@ -1,17 +1,18 @@
 # FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
-define postfix::postmapfile($named) {
-        exec { "postmap_${named}":
-                command     => "/usr/sbin/postmap /etc/postfix/${named}",
-                refreshonly => true,
-                require     => [
-                            File["/etc/postfix/${named}"],
-                            Package['postfix']],
-        }
-        file { "/etc/postfix/${named}":
-                ensure  => present,
-                mode    => '0644',
-                content => template("postfix/etc/postfix/${named}.erb"),
-                notify  => Exec["postmap_${named}"],
-        }
+define postfix::postmapfile(
+  $content,
+) {
+  $filename = "/etc/postfix/${title}"
+
+  Package['postfix']
```

Comment:
I think the Package here is not needed, we already have https://github.com/alphagov/govuk-puppet/blob/master/modules/postfix/manifests/init.pp#L46, maybe it was residual